src/BlockTypes/MiniCartContents.php

```diff
@@ -130,7 +136,7 @@ function( $property ) {
 			);
 
 			if ( ! empty( $properties ) ) {
-				$parsed_style .= $style['selector'] . '{' . PHP_EOL;
+				$parsed_style .= $selector . '{' . PHP_EOL;
```

Comment:
...and remove `PHP_EOL` here and the lines below.

I think we don't need to nice-format CSS if we are printing it directly into the page and instead should prioritize making it shorter. But not a big deal, so leaving the final decision to you.